google/cloud/spanner/instance_admin_connection.cc

```diff
@@ -298,7 +302,9 @@ class InstanceAdminConnectionImpl : public InstanceAdminConnection {
   std::unique_ptr<RetryPolicy const> retry_policy_prototype_;
   std::unique_ptr<BackoffPolicy const> backoff_policy_prototype_;
   std::unique_ptr<PollingPolicy const> polling_policy_prototype_;
+  std::unique_ptr<BackgroundThreads> background_threads_;
```

Comment:
Ditto regarding thread pools and ownership cycles.

Reply:
Done.